src/vectors/angular.rs

```diff
@@ -13,6 +13,10 @@ use crate::{
 /// The cosine similarity is defined as the dot product of the two vectors
 /// divided by the product of their magnitudes.
 ///
+/// Computing this distance with vectors of differing or zero dimensionality is
+/// nonsensical - attempting to do so will panic in debug mode, and may silently
+/// give incorrect values in release mode.
+///
```

Comment:
These docs should be present exactly once in the docs at the top of the `vectors` module.